src/plugins/intel_cpu/src/nodes/rope.cpp

```diff
@@ -252,15 +252,26 @@ struct RoPE::RoPEExecutorChatGLM : public RoPE::Executor {
         jcp.dst_prc = precision_of<T>::value;
         jcp.rotary_ndims = config.rotary_ndims;
         jcp.interleave = true;
-        jcp.mix_cos_sin = true;
+        // if use precomputed rope cache then it's mixed
+        // otherwise rope will have separate cos/sin inputs
+        jcp.mix_cos_sin = config.use_rope_cache;
         m_rotaryKernel = createJitKernel(jcp, true);
     }
 
     void execute([[maybe_unused]] const dnnl::stream& strm,
                  const std::vector<MemoryPtr>& inputs,
                  const std::vector<MemoryPtr>& outputs) override {
         ov::intel_cpu::PlainTensor t_src(inputs[0]);
-        ov::intel_cpu::PlainTensor t_cos_sin(inputs[1]);
+        ov::intel_cpu::PlainTensor t_cos;
+        ov::intel_cpu::PlainTensor t_sin;
```

Comment:
These two objects are initialized only when `m_config.use_rope_cache` is false. Does it make sense to put their instance and initialization close to their usage. To L314?